docs/conf.py

```diff
@@ -358,10 +358,10 @@ def _get_rst_filepath_from_path(filepath: pathlib.Path):
 
 # Theme configuration
 if PACKAGE_NAME.startswith("apache-airflow-providers-"):
-    # Only hide hidden items for providers. For Chart and Airflow we are using the approach where
-    # TOC is hidden but sidebar still shows the content (but we are not doing it for providers).
-    html_theme_options: dict[str, Any] = {"hide_website_buttons": True, "sidebar_includehidden": False}
+    # Hide website buttons for providers
+    html_theme_options: dict[str, Any] = {"hide_website_buttons": True}
 else:
+    # For chart we hide website buttons but also do not include hidden TOC contents
```

Comment:
Confused with the comment here. The comment says do not include hidden TOC contents, but the value is set to True below.

Reply:
Oh. yeah. Actually after this change we can just have exactly the same theme config for all packages (airflow, chart, providers because they are all the same 🤦